configuration/reissue-internal-certificates.adoc

```diff
@@ -13,17 +13,35 @@ You can view the certificate expiration dates by selecting *Platform Configurati
 
 //Add link to role based permissions and resources
 
-//reissue internal certificates for Central
-include::modules/reissue-internal-certificates-central.adoc[leveloffset=+1]
+//Internal certificate authority rotation for RHACS
+include::modules/internal-certificate-authority-rotation-for-rhacs.adoc[leveloffset=+1]
 
-//Restart Central
-include::modules/restart-central-container.adoc[leveloffset=+2]
+//Supported installation methods
+include::modules/supported-installation-methods.adoc[leveloffset=+2]
 
-//reissue internal certificates for Scanner
-include::modules/reissue-internal-certificates-scanner.adoc[leveloffset=+1]
+//Required user actions
+include::modules/required-user-actions.adoc[leveloffset=+2]
```

Comment:
Looking at the rendered page, it feels like it is starting to contain too much information. What do you think about making a distinct section just for CA rotation?